local/client/conn/conn.go

```diff
@@ -0,0 +1,66 @@
+package conn
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"time"
+)
+
+// A Dialer can dial a connection to the Scoot server.
```

Comment:
Cool I've just been doing /* */ but that's a Java / Scala habit happy to change to whatever is more standard or works best with godoc